ramalama/cli.py

```diff
@@ -167,55 +167,70 @@ def configure_arguments(parser):
+        "--nocontainer",
+        dest="container",
+        default=config.get("nocontainer"),
+        action="store_false",
+        help="""do not run RamaLama in the default container.
+The RAMALAMA_IN_CONTAINER environment variable modifies default behaviour.""",
     )
-    parser.add_argument("--dry-run", dest="dryrun", action="store_true", help=argparse.SUPPRESS)
     parser.add_argument(
         "--engine",
         dest="engine",
         default=config.get("engine"),
         help="""run RamaLama using the specified container engine.
 The RAMALAMA_CONTAINER_ENGINE environment variable modifies default behaviour.""",
     )
-    parser.add_argument(
-        "--gpu",
-        dest="gpu",
-        default=False,
-        action="store_true",
-        help="offload the workload to the GPU",
-    )
-    parser.add_argument(
-        "--image",
-        default=config.get("image"),
-        help="OCI container image to run with the specified AI model",
-    )
-    parser.add_argument(
-        "--nocontainer",
-        dest="container",
-        default=config.get("nocontainer"),
-        action="store_false",
-        help="""do not run RamaLama in the default container.
-The RAMALAMA_IN_CONTAINER environment variable modifies default behaviour.""",
-    )
+
+
+def add_runtime_arguments(parser):
+    """Add runtime-related arguments to the parser."""
     parser.add_argument(
         "--runtime",
         default=config.get("runtime"),
         choices=["llama.cpp", "vllm"],
         help="specify the runtime to use; valid options are 'llama.cpp' and 'vllm'",
     )
     parser.add_argument(
-        "--store",
-        default=config.get("store"),
-        help="store AI Models in the specified directory",
+        "--image", default=config.get("image"), help="OCI container image to run with the specified AI model"
+    )
+    parser.add_argument(
+        "--privileged", dest="privileged", action="store_true", help="give extended privileges to container"
+    )
+    parser.add_argument("--gpu", dest="gpu", action="store_true", help="offload the workload to the GPU")
+
+
+def add_debug_arguments(parser):
+    """Add debug-related arguments to the parser."""
+    parser.add_argument("--debug", action="store_true", help="display debug messages")
+
+
+def add_dryrun_arguments(parser):
+    """Add dry run arguments to the parser."""
+    parser.add_argument(
+        "--dryrun", dest="dryrun", action="store_true", help="show container runtime command without executing it"
     )
+
+
+def add_store_arguments(parser):
+    """Add store-related arguments to the parser."""
+    parser.add_argument("--store", default=config.get("store"), help="store AI Models in the specified directory")
+
+
+def add_version_arguments(parser):
+    """Add version-related arguments to the parser."""
     parser.add_argument("-v", "--version", dest="version", action="store_true", help="show RamaLama version")
 
 
+def configure_arguments(parser):
```

Comment:
This should have a more specific name.

configure_runtime_container_arguments?